action.yml

```diff
@@ -14,8 +14,14 @@ inputs:
   tool-path:
     description: "Path to the data tool files (e.g. migration)."
     required: true
+  data-dictionary-app-id:
```

Comment:
You can drop "data-" from all of these, since everything here is in the same scope. I'd leave "dictionary-" though, otherwise it would get confusing which app/repo.